mmv1/products/cloudfunctions2/Function.yaml

```
@@ -91,6 +91,26 @@ examples:
     ignore_read_extra:
       - 'build_config.0.source.0.storage_source.0.object'
       - 'build_config.0.source.0.storage_source.0.bucket'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloudfunctions2_scheduler_auth'
+    primary_resource_id: 'function'
+    vars:
+      bucket_name: 'gcf-source'
+      service_account: 'gcf-sa'
+      function: 'gcf-function'
+      zip_path: 'function-source.zip'
+    test_env_vars:
+      project: :PROJECT_NAME
+    test_vars_overrides:
+      zip_path: '"./test-fixtures/cloudfunctions2/function-source.zip"'
+      primary_resource_id: '"terraform-test"'
+      location:
+        '"us-central1"'
+        # ignore these fields during import step
+    ignore_read_extra:
+      - 'build_config.0.source.0.storage_source.0.object'
+      - 'build_config.0.source.0.storage_source.0.bucket'
+    skip_test: true 
```

```suggestion
    skip_test: true
```
Pre-build validation failed due to 
```
repo/mmv1/products/cloudfunctions2/Function.yaml
  Error: 113:20 [trailing-spaces] trailing spaces
```
The suggested change should fix it. Sorry about it